easybuild/easyconfigs/s/Subread/Subread-2.0.0-foss-2018b.eb

```diff
@@ -0,0 +1,28 @@
+# Contribution from the Crick HPC team
+# modified for version 2.0.0 by Felix Schmitt
+
+easyblock = 'MakeCp'
+
+name = 'Subread'
+version = '2.0.0'
+
+homepage = 'http://%(namelower)s.sourceforge.net/'
+description = """High performance read alignment, quantification and mutation discovery"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
@sciroc To make @migueldiascosta suggested change a bit more concrete, this can/should be changed to:

```python
toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}

Reply:
@migueldiascosta @boegel 
Thanks for your feedback. Changed toolchain to GCC 7.3.0-2.30 and updated PR.

Reply:
@sciroc great, thanks

you now have both files in this PR, you need to delete the `foss` one (e.g., with https://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html#deleting-easyconfig-files-or-patches)

Reply:
@migueldiascosta Done, thanks!